nextflow_schema.json

```
@@ -17,16 +17,39 @@
                     "format": "file-path",
                     "exists": true,
                     "schema": "assets/schema_input.json",
-                    "mimetype": "text/csv",
-                    "pattern": "^\\S+\\.csv$",
-                    "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/pathogensurveillance/usage#samplesheet-input).",
+                    "mimetype": "text/tsv",
+                    "pattern": "^\\S+\\.[ct]sv$",
+                    "description": "Path to comma/tab-separated file containing information about samples.",
+                    "help_text": "This TSV/CSV has one row per samples and contains information such as the location of input files, sample ID, labels, etc. Use this parameter to specify its location. See [the usage page](https://nf-co.re/pathogensurveillance/dev/docs/usage/) for details on formatting this file.",
+                    "fa_icon": "fas fa-file-csv"
+                },
+                "reference_data": {
+                    "type": "string",
+                    "format": "file-path",
+                    "mimetype": "text/tsv",
+                    "pattern": "^\\S+\\.[ct]sv$",
+                    "schema": "assets/schema_input.json",
+                    "description": "Path to comma-separated file containing information about samples.",
+                    "help_text": "This TSV/CSV has one row per reference and contains information such as the location of input files, reference ID, labels, etc. Use this parameter to specify its location. See [the usage page](https://nf-co.re/pathogensurveillance/dev/docs/usage/) for details on formatting this file.",
```

```suggestion
                    "help_text": "This TSV/CSV has one row per reference and contains information such as the location of input files, reference ID, labels, etc. Use this parameter to specify its location. See [the usage page](https://nf-co.re/pathogensurveillance/usage/) for details on formatting this file.",
```
better to not point to a specific version